universal-login-sdk/lib/observers/DeploymentObserver.ts

```diff
@@ -16,7 +17,7 @@ export class DeploymentObserver extends ObserverRunner {
   }
 
   startAndSubscribe(futureContractAddress: string, callback: OnContractDeployed) {
-    ensure(!this.isRunning(), Error, 'DeploymentObserver is waiting for contract deployment. Stop observer to cancel waiting');
+    ensure(!this.isRunning(), DeploymentObserverConfilct);
```

Comment:
`DeploymentObserverConfilct` -> `ConcurrentDeployment`

Reply:
DeploymentObserverConfilct and BalanceObserverConflict merge into one type error?

Reply:
Sounds good, same reason, same error.